File: Tasks/Python Work/arithmetic_operations.py
number_one = int(input("Enter the first number: "))
number_two = int(input("Enter the second number: "))
number_three = int(input("Enter the third number: "))

sum = number_one + number_two + number_three
average = sum/3
product = number_one * number_two * number_three
smallest = min(number_one, number_two, number_three) 
largest = max(number_one, number_two, number_three) 

print("The sum is", sum)
print("The average is", average)
print("The product is", product)
print("The smallest number is", smallest)
print("The largest number is", largest)

File: Tasks/Java Work/100 Tasks/ArithmeticOperations.java
import java.util.Scanner;
	public class ArithmeticOperations{
		public static void main(String[] args){
		Scanner input = new Scanner(System.in);

System.out.println("Enter the first number:");
double numberOne = input.nextDouble();

System.out.println("Enter the second number:");
double numberTwo = input.nextDouble();

double sum = 0;
double difference = 0;
double product = 0;
double quotient = 0;

sum = numberOne + numberTwo;
difference = numberOne - numberTwo;
product = numberOne * numberTwo;
quotient = numberOne/numberTwo;

System.out.println("The sum = " + sum);
System.out.println("The difference = " + difference);
System.out.println("The product = " + product);
System.out.println("The quotient = " + quotient);
}
}



